include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -233,11 +280,25 @@ class SplineBuilder
                 bsplines_type>(batched_interpolation_domain(), spline_domain());
     }
 
+    /**
+     * @brief Get the whole domain on which spline coefficients are defined, with dimension of interest contiguous.
+     *
+     * This is used internally because of solvers limitation and because it may be beneficial to computation performance.
```

Comment:
No my bad the dimension of interest is coalescent with Ginkgo

Reply:
![image](https://github.com/CExA-project/ddc/assets/25363473/ccbe04a5-78f8-4017-85ac-8f3f39da5c71)
I don't understand what "coalescent" means in the context of a dimension of interest.

Reply:
Yes looking back at it I realize this terminology is not standard. My intention was to refer to coalesced memory access. It seems that `leading dimension` is standard for 2D arrays. I'm not sure for ND arrays but as long as "leading" clearly means there is a unique one I think we could use this here ?

Reply:
> My intention was to refer to coalesced memory access

This term isn't much clearer for me :sweat_smile: . I googled it and it seems [I am not the only one to find this unclear. It may also be cuda specific terminology](https://stackoverflow.com/questions/5041328/in-cuda-what-is-memory-coalescing-and-how-is-it-achieved).

I have heard `leading dimension` several times before. I agree that this is clear for ND arrays.